Software/Python/brickpi3.py

```diff
@@ -590,9 +592,9 @@ def transact_i2c(self, port, Address, OutArray, InBytes):
         outArray = [self.SPI_Address, message_type, Address, InBytes]
         self.I2CInBytes[port_index] = InBytes
         OutBytes = len(OutArray)
-        if(OutBytes > 16):
-            outArray.append(16)
-            for b in range(16):
+        if(OutBytes > self.I2C_LENGTH_LIMIT):
```

Comment:
Hey @mattallen37 just wanted to make sure you saw this was going on.  If you meant to add this into the brickuber, then happy to approve.

Reply:
Good catch. That was an older commit, used in development, but not yet ready for release. I will change the limit back to 16 for now (which is the current firmware limit).

Reply:
It has now been changed back to 16. Should be good now.